classes/main.rb

```diff
@@ -1,5 +1,7 @@
 require './teacher'
 require './student'
 require './person'
+require './book'
 require './capitalize_decorator'
 require './trimmer_decorator'
+require './rental'
```

Comment:
- [ ] Kindly ensure your main.rb file has a method that creates an action to start your application, currently only importing classes to your main.rb file, here is an error ma getting when trying to run your appication
![image](https://user-images.githubusercontent.com/61408860/180268325-9e95027b-2028-4e5d-91c7-618bd8c30ed9.png)

Reply:
> * [x]  Kindly ensure your main.rb file has a method that creates an action to start your application, currently only importing classes to your main.rb file, here is an error ma getting when trying to run your appication
>   ![image](https://user-images.githubusercontent.com/61408860/180268325-9e95027b-2028-4e5d-91c7-618bd8c30ed9.png)

Thank you @danmainah , I made the entry point right after submitting in my recent commit. Please take a look again